about.css

```diff
@@ -0,0 +1,1065 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+:root {
+  --color-white: #fff;
+  --color-secondary: #ec5242;
+  --color-secondaryDeep: #e2220c;
+  --color-lightblack: #272a31;
+  --color-lightgrey: #d3d3d3;
+  --color-black: #000;
+}
+
+body {
+  width: 100%;
+  font-family: "Lato", sans-serif;
+  font-size: 1.1rem;
+}
```

Comment:
- [ ] You have some gap to the right side of your website, please kindly fix that 🙃 🙃 

Your design

<img width="316" alt="Screenshot 2023-03-24 at 21 05 58" src="https://user-images.githubusercontent.com/74501523/227606433-48f92114-524e-4aef-82cb-0ec2ac26eef8.png">